frontend/src/components/Icons.tsx

```diff
@@ -57,6 +56,36 @@ export const CloseIcon = ({
   );
 };
 
+/** Bento button that inherits the text color of its container */
+export const BentoIcon = (
+  props: SVGProps<SVGSVGElement> & { alt?: string }
+) => {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      role="img"
+      aria-label={props.alt}
+      aria-hidden={props.alt === ""}
+      fill="none"
```

Comment:
Nit, but I think this is redundant with the `style`? (Just noticed `<CogWheel>` has this too, which I probably did :sweat_smile: )

```suggestion
```